docs/libindy/macos-apple.md

```diff
@@ -12,22 +12,22 @@
 The first thing we'll do is install OpenSSL. Since Apple replaced OpenSSL with their own version of LibreSSL, we'll need to install it. Also, we need to install a specific version of OpenSSL that is compatible with Apples architecture. After the installation, we need to link it, so that it overrides the default openssl command (we have not noticed any issues with overriding this command, but be cautious).
 
 ```sh
-curl https://raw.githubusercontent.com/rbenv/homebrew-tap/e472b7861b49cc082d1db0f66f265368da107589/Formula/openssl%401.0.rb -o openssl@1.0.rb
+brew install openssl@1.1 # possibly already installed on your system
 
-brew install ./openssl@1.0.rb
+brew link openssl@1.1 --force
 
-rm -rf ./openssl@1.0.rb
+echo 'export PATH="/opt/homebrew/opt/openssl@1.1/bin:$PATH"' >> ~/.zshrc
```

Comment:
Not the end of the world but not everyone uses zsh. Maybe bash, ash ... or even fish?

Reply:
True, but since ZSH is macOs native I think anyone installing a custom shell will know what to do here...

Reply:
alrighty!